### PR TITLE
sqlcmd 1.1.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.<details>
  <summary>prerelease notes</summary>
  <pre>

## Changes:

* d1965d109bab32a1f8ab087fae14ff704df40657 usage formatting (#360)
* c60bbadd7c97e6c4d8e2c12303637a429689a010 Add localization settings files for loc pipeline (#359)
* 00fb6093646ad6befc4c40a761e314c44369ef9c Wrap pending user visible content with localizer routines (#325) [ #298 ]
* a63d168ca4b0c6d530732a11614e58d6ba0ebfd7 feat: Add -A switch (#361)
* 17426d60f65e52d996376f4491deaf8cc37bbc8d removing extra screenwidth entry (#358)
* 1f617dd471867aa6d32f296973c1c1bd582191a8 removing extra error and usage print (#352)
* 4514e86b18af1dc3e5b7e73105ebb7edd2d248e6 Mark release as pre-release (#357)
* efaa12ebb0cf5f917314f6f869795a16b8044fc1 help text message resumed (#353)
* 13a1680a01b37affa1c23dc9381f7c38d0c68e47 Kong to Cobra Migration  (#208)
* d754626a4e9bac4ec420aa467c19554b359a697f Do not read basic auth info if auth type is other (#340)
<details><summary><b>See More</b></summary>

* 16095e40a0892b9a664eec6e513d928cf92f0dda Bump github.com/docker/distribution from 2.8.1+incompatible to 2.8.2+incompatible (#338)
* 164f8e10fa7ac6a3784b29cf6901970462e95ded implement :xml (#336)

This list of changes was [auto generated](https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=201825&view=logs).</details></pre>
</details>

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  Minitest::Assertion: Expected /sqlcmd:\ 1\.1\.0/ to match "1.1.0\nLegal docs and information: aka.ms/SqlcmdLegal\nThird party notices: aka.ms/SqlcmdNotices\n".
```

**Review by p-linnane — DISMISSED** [review]

**Comment by p-linnane** [comment]
This is marked as a pre-release: https://github.com/microsoft/go-sqlcmd/releases

File: Formula/sqlcmd.rb [hunk 1]
```diff
@@ -25,9 +30,9 @@ def install
   end
 
   test do
-    out = shell_output("#{bin}/sqlcmd -S 127.0.0.1 -E -Q 'SELECT @@version'", 1)
+    out = shell_output(bin/"sqlcmd -S 127.0.0.1 -E -Q 'SELECT @@version'", 1)
```
Comment (carlocab) [inline Formula/sqlcmd.rb:33]:
```suggestion
    out = shell_output(bin/"sqlcmd -S 127.0.0.1 -E -Q 'SELECT @@version'", 1)
```
This command is not a pathname, so we should technically interpolate here.

File: Formula/sqlcmd.rb [hunk 2]
```diff
@@ -25,9 +30,9 @@ def install
...
     assert_match "connection refused", out
 
-    assert_match "sqlcmd: #{version}", shell_output("#{bin}/sqlcmd --version")
+    assert_match version.to_s, shell_output(bin/"sqlcmd --version")
```
Comment (carlocab) [inline Formula/sqlcmd.rb:36]: Same here.

File: Formula/sqlcmd.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/sqlcmd.rb]: 👍

File: Formula/sqlcmd.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/sqlcmd.rb]: 👍

**Review by p-linnane — APPROVED** [review]

**Pull request merged**